### PR TITLE
created home page

### PR DESCRIPTION
**Pull request by sagargse**
Created the Home page 
image

### CODE REVIEW

**Comment by Sharmaryan** [comment]
homes page looks awesome

File: home-page/index.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
```
Comment (Sharmaryan) [inline home-page/index.html:8]: although it is not mandatory but you can use meta tags for author and description as well

**Review by Sharmaryan — APPROVED** [review]
nice use of variables, and UI also looks good

File: home-page/index.html (hunk 1 above)
Comment (sagargse) [reply home-page/index.html:8]: ok will consider.

File: cssfiles/footer.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.footer-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--primary-color);
+    color: var(--primary-text-color);
+    padding: 1rem;
```
Comment (Prabhav-Sharma) [inline cssfiles/footer.css:8]: Can use globally defined variables for spacing properties like margin, padding, and gap to maintain consistency throughout the website

Comment (sagargse) [reply cssfiles/footer.css:8]: ok man will consider that.

**Pull request merged**